client/src/pages/Admins/AdminFinance/UserSelector/UserSelector.js

```diff
@@ -30,7 +30,7 @@ class UserSelector extends React.Component {
             this.setState({ indexOfActiveItem: index });
         }
         if (e.keyCode === 13 && showSuggestions) {
```

Comment:
You are checking if e.keyCode is 13 on line 41 as well. Couldn't you check if `e.keyCode === 13` once in an outside if statement?

Reply:
Restructured that part, should look neater now